features/create-conversation/create-conversation-screen.tsx

```diff
@@ -91,30 +70,56 @@ import { NavigationParamList } from "../Navigation/Navigation";
  */
 type NavigationProps = NativeStackNavigationProp<
   NavigationParamList,
-  "NewConversation"
+  "CreateConversation"
 >;
 
-export default function NewConversation({
+export function CreateConversationScreen({
   navigation,
 }: {
   navigation: NavigationProps;
 }) {
   const { theme, themed } = useAppTheme();
   const [conversationCreationMode, setConversationCreationMode] =
     useState<ConversationVersion>(ConversationVersion.DM);
-  // const navigation = useNavigation();
 
-  const handleBack = useCallback(() => {
-    logger.debug("[NewConversation] Navigating back");
-    navigation.goBack();
-  }, [navigation]);
+  const [
+    searchQueryState,
+    /*weird name becuase I'm not exactly sure what the ref below does so being expliti as state vs ref*/ setSearchQueryState,
```

Comment:
Not sure this comment but you shouldn't be able to add yourself to a group or a dm.